packages/recipes/src/index.ts

```diff
@@ -63,7 +69,12 @@ export function recipe<Variants extends VariantGroups>(
 
   const config: PatternResult<Variants> = {
     defaultClassName,
+    rawDefaultClassName: defaultClassName.split(' ')[0],
```

Comment:
When you have an array of "string" class names (i.e. `base: ['one', 'two']`) the `style()` function computes the final class names internally and concats all of them together. I make an assumption that the first one is the generated one